bootstrap.py

```diff
@@ -174,6 +174,17 @@ def get_bootstrap_rpm():
     exec_failexit("rpm -Uvh http://%s/pub/katello-ca-consumer-latest.noarch.rpm" % options.foreman_fqdn)
 
 
+def disable_rhn_plugin():
+    rhnpluginconfig = SafeConfigParser()
+    rhnpluginconfig.read('/etc/yum/pluginconf.d/rhnplugin.conf')
```

Comment:
what happens if the file is not present? `SafeConfigParser.read()` will survive, but the `get()`?
I think we should wrap this into `if os.path.exists('/etc/yum/pluginconf.d/rhnplugin.conf')`, no?